apps/workflow-cf.yml.j2

```diff
@@ -67,11 +67,13 @@ Resources:
           {% for command in task['command'] %}
           - {{ command }}
           {% endfor %}
+        {% if task.get('secrets', []) %}
```

Comment:
I believe you could get rid of the `if` block entirely, and edit the `for` statement to be: `for secret in task.get('secrets', [])`